meshio/med_io.py

```diff
@@ -9,14 +9,20 @@
 '''
 import numpy
 
-
+# [code_aster_src]/catalo/cataelem/Commons/mesh_types.py
```

Comment:
An actual HTTPS link would be nice.

Reply:
Ok I will replace it by `https://bitbucket.org/code_aster/codeaster-src/src/`

Reply:
With a location in code please. Just the repo isn't enough.